x/ratelimit/keeper/path.go

```diff
@@ -0,0 +1,78 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/proto"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
+)
+
+func FormatPathId(denom string, channelId string) string {
+	return fmt.Sprintf("%s-%s", denom, channelId)
+}
+
+func (k Keeper) AddPath(ctx sdk.Context, path types.Path) string {
+	store := ctx.KVStore(k.storeKey)
+	prefixStore := prefix.NewStore(store, types.KeyPrefix(types.PathKey))
```

Comment:
It might make more sense to change `PathKey` to `PathKeyPrefix`

Reply:
agreed, updated